packages/yourloops/test/integration/assert/patient-statistics.assert.ts

```diff
@@ -41,7 +41,7 @@ export const checkPatientStatistics = async () => {
   expect(patientStatistics.getByTestId('cbg-percentage-bar-chart')).toHaveTextContent('Avg. Daily Time In Range4h 48m20%4h 48m20%4h 48m20%4h 48m20%4h 48m20%<5454-7070-180180-250>250mg/dL')
   expect(patientStatistics.getByTestId('cbg-standard-deviation-stat')).toHaveTextContent('Standard Deviation (24-224)mg/dL100')
   expect(patientStatistics.getByTestId('cbg-mean-stat')).toHaveTextContent('Avg. Glucose (CGM)mg/dL124')
-  expect(patientStatistics.getByTestId('container-insulin-stats')).toHaveTextContent('Avg. Daily Total Insulin2.1UBolus1 U48%Basal1.1 U52%Weight72kgRatio Dose0.03U/kg')
+  expect(patientStatistics.getByTestId('container-insulin-stats')).toHaveTextContent('Avg. Daily Total Insulin5.4UBolus1 U19%Basal4.4 U81%Weight72kgRatio Dose0.08U/kg')
```

Comment:
Note: This is likely to have a conflict with the release branch... Can you make compare your branch to ppd and check if you have the same insulin values?